sensor/kubernetes/complianceoperator/dispatchers/complianceoperatorremediations.go

```diff
@@ -24,20 +25,19 @@ func NewRemediationDispatcher() *RemediationDispatcher {
 
 // ProcessEvent processes a compliance operator remediation
 func (c *RemediationDispatcher) ProcessEvent(obj, _ interface{}, action central.ResourceAction) *component.ResourceEvent {
-	// TODO(sbaumer): enable dispatcher when central pipeline is added
-	if !centralcaps.Has(centralsensor.ComplianceV2Integrations) || true {
+	if !centralcaps.Has(centralsensor.ComplianceV2Integrations) || !features.ComplianceRemediationV2.Enabled() {
```

Comment:
If there is any chance this may stick for 4.5, we should use centralcaps here instead of the feature flag.  If this is temporary and will be removed before 4.5, we should write a ticket and add a `TODO`

Reply:
Is this a pattern that we do not use feature flags in Sensor? Or what is the advantage of the capability?

My thought is that the feature should be disabled until we explicitly enable it. In this case Sensor would always start collecting remediations, even if the feature flag is disabled at Central.

Reply:
Since sensor could be any number of versions on any number of clusters, it has become the preference as of late to avoid feature flags for central features in sensor.  Central can tell sensor the messages it understands and sensor can send only those.  

TBH, I think a feature flag for this is overkill in general, so it probably doesn't matter too much.

Reply:
Added the caps.